exe/genv-envs

```diff
@@ -17,17 +17,27 @@ QUERIES = {
     'config.gpu_memory': lambda env: env["config"].get("gpu_memory"),
 }
 
+
 def factory() -> Dict:
+    """
+    Create a 'empty' state object for genv environments.
+    :return: The state object (dict of dicts)
+    """
     return dict(
         envs={}
     )
 
+
 def cleanup(
         envs: Envs,
-        poll_pid: Callable[[int], bool]=genv.poll_pid,
-        poll_kernel: Callable[[str], bool]=genv.poll_kernel) -> None:
+        poll_pid: Callable[[int], bool] = genv.poll_pid,
+        poll_kernel: Callable[[str], bool] = genv.poll_jupyter_kernel) -> None:
     """
-    Clean up processes, kernels and environments.
+    Clean up processes, kernels and environments connected to the given environment.
```

Comment:
btw wdym with "connected to the given environment"?

Reply:
My mistake. I think that I mixed up something. I will restore the original content.